buyer-signing-service/test/routes/sign/testEndpointInput.js

```diff
@@ -0,0 +1,38 @@
+import request from 'supertest';
+import app from '../../../src/app';
+
+const createUnprocessableEntityError = (prop, message) => ({
+  message,
+  statusCode: 422,
+  error: 'Unprocessable Entity',
+  errors: [`Field '${prop}' is required`],
+});
+
+export function testEndpointInput(endpoint, inputParams, {
```

Comment:
Great tool! Do you think it can work against endpoints that include swagger validations?

Reply:
@Meketh